app/internal/email.py

```diff
@@ -52,6 +52,49 @@ def send(
     return True
 
 
+def send_email_to_event_participants(
+        session: Session, event_id: int,
+        title: str, content: str,
+        background_tasks: BackgroundTasks = BackgroundTasks
+) -> bool:
+    """This function is being used to send emails in the background.
+    it uses the main elements of the func writen above.
+    It takes an event and a user and it sends the event to the user.
+    Args:
+        session(Session): The session to redirect to the database.
+        title (str): Title of the email that is being sent.
+        event_id (int): Id number of the event that is used.
+        content (str): body of email sent.
+        background_tasks (BackgroundTasks): Function from fastapi that lets
+            you apply tasks in the background.
+    Returns:
+        bool: Returns True if emails were sent, False if none.
+    """
+    event_owner = session.query(Event.owner).filter(id == event_id).first()
+    if event_owner != get_current_user(session):
+        return False
+    # makes sure only event owner can send an email via this func.
+    mailing_list = session.query(User.email).join(
+        UserEvent, User.id == UserEvent.user_id
+        ).filter(
+            event_id == event_id).all()
+    valid_mailing_list = list(filter(verify_email_pattern, mailing_list))
+    if not valid_mailing_list:
+        return False
+    # making sure app doesn't crash if emails are invalid
+    event = session.query(Event).get(event_id)
+    subject = f"{event.title}: {title}"
+    recipients = valid_mailing_list
+    body = content
+    for r in recipients:
+        print(r)
```

Comment:
remove

Reply:
thank you! will be removed on next commit